.github/workflows/ci.yml

```diff
@@ -79,25 +79,33 @@ jobs:
         if: matrix.os != 'macos-latest'
         run: make test
 
+      # Store a cnd ${{ matrix.os }} archive that contains the cnd binary
       - name: Upload cnd ${{ matrix.os }} binary
-        if: matrix.os == 'ubuntu-latest'
         uses: actions/upload-artifact@v1
         with:
-          name: cnd
+          name: cnd-${{ matrix.os }}
           path: target/debug/cnd
 
-
   e2e_test:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest]
+        include:
+          - os: ubuntu-latest
+            binary-suffix: ""
```

Comment:
I don't think we are using this binary-suffix for anything. That seems to be a copy-pasta artifact from create-comit-app.
We also don't need the `include` part because the matrix above describes all the combinations already.

Reply:
I think the same applies for some other job further above.

Reply:
done